### ISSUE TITLE
Switch to HTTPS on r754

### ISSUE THREAD

**Issue by yellowhornet** [ignacionelson/ProjectSend#504]
I am trying to use HTTPS with a projectsend site.
I change the System URI box on the options menu and choose Update all options and I lose all formatting (see picture)
![Uploading lossofformatting.PNG…]()
I have tried following suggestions on other posts #65 etc but the files that they suggest do not seem to contain the fields 
This is a clean Centos 7 server with no other web servers running on it.

**Comment by ignacionelson** [comment]
Hello!
Can you please re upload the image? It's not showing up.
Thanks!

**Comment by yellowhornet** [comment]
![lossofformatting](https://user-images.githubusercontent.com/17897762/34086968-c7f9fcf6-e3eb-11e7-8040-c12dec562ec3.PNG)

**Comment by yellowhornet** [comment]
I have completed a new installation as the original one was running extremly slowly (eg taking 1-2 minutes to switch menus) I still get the same error as above and I know have a new problem of not being able to upload files at all
![upload1](https://user-images.githubusercontent.com/17897762/34087018-374f3e9a-e3ec-11e7-8057-d0bda5743ff0.PNG)
![upload2](https://user-images.githubusercontent.com/17897762/34087019-377e4410-e3ec-11e7-8696-e1831e31e904.PNG)
![upload3](https://user-images.githubusercontent.com/17897762/34087020-37ac950e-e3ec-11e7-80fa-f8ef1700c477.PNG)

**Comment by coldragon** [comment]
Same prob here

**Comment by tman1982** [comment]
I am seeing the exact same issue, except what I did was setup my SSL Cert and redirect in Apache. Then this started. I was able to get to the options and change the SITE URI to 'https' but now when I login I get the below text. Thoughts? (The xxxxx is me blanking out the actual public URL) If you click on the URL you get NOT FOUND

{"status":"success","message":"
×<\/a>Login success. Redirecting...<\/div>","location":"http:\/\/xxxxxxxxx.xxxxxxx.com\/home.php"}
